vtex/ab-tester
# Make AB-Tester log errors on Colossus

#### What problem is this solving?
Every error should be logged on Colossus and this makes them searchable on Splunk.

#### How should this be manually tested?
Link AB-Tester and then look for a name of a workspace that doesn't. Then try to initialize the A/B test for this workspace. That will throw a error meaning that this workspace doesn't exists and saying what should be searched in Splunk to see the error. Then search it and see there the error.

#### Types of changes
